sample-templates/query-assist-agent-claude.json

```diff
@@ -1,90 +1,108 @@
-      "3.0.0"
-    ]
+    "compatibility": ["2.13.0", "3.0.0"]
   },
   "workflows": {
     "provision": {
+      "user_params": {},
       "nodes": [
         {
-          "id": "create_openai_connector",
+          "id": "create_claude_connector",
           "type": "create_connector",
+          "previous_node_inputs": {},
           "user_inputs": {
-            "name": "OpenAI Chat Connector",
-            "description": "The connector to public OpenAI model service for GPT 3.5",
             "version": "1",
-            "protocol": "http",
-            "parameters": {
-              "endpoint": "api.openai.com",
-              "model": "gpt-3.5-turbo"
-            },
-            "credential": {
-              "openAI_key": "PUT_YOUR_API_KEY_HERE"
-            },
+            "name": "Claude instant runtime Connector",
+            "protocol": "aws_sigv4",
+            "description": "The connector to BedRock service for Claude model",
             "actions": [
               {
-                "action_type": "predict",
+                "headers": {
+                  "x-amz-content-sha256": "required",
+                  "content-type": "application/json"
+                },
                 "method": "POST",
-                "url": "https://${parameters.endpoint}/v1/chat/completions"
+                "request_body": "{\"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
+                "action_type": "predict",
+                "url": "https://bedrock-runtime.us-west-2.amazonaws.com/model/anthropic.claude-instant-v1/invoke"
               }
-            ]
+            ],
+            "credential": {
+                "access_key": "<put your access key here>",
+                "secret_key": "<put your secret key here>"
+            },
+            "parameters": {
+              "region": "us-west-2",
+              "endpoint": "bedrock-runtime.us-west-2.amazonaws.com",
+              "content_type": "application/json",
+              "auth": "Sig_V4",
+              "max_tokens_to_sample": "8000",
+              "service_name": "bedrock",
+              "temperature": "0.0001",
+              "response_filter": "$.completion",
+              "anthropic_version": "bedrock-2023-05-31"
+            }
           }
         },
         {
-          "id": "register_openai_model",
+          "id": "register_claude_model",
           "type": "register_remote_model",
           "previous_node_inputs": {
-            "create_openai_connector": "connector_id"
+            "create_claude_connector": "connector_id"
           },
           "user_inputs": {
-            "name": "openAI-gpt-3.5-turbo",
-            "deploy": true
+            "description": "Claude model",
+            "deploy": true,
+            "name": "claude-instant",
+            "guardrails": {
+              "type": "",
```

Comment:
No default type here?

Reply:
What do you want me to put there? Any non-null string works.

Reply:
Changed type to `local_regex` per the documentation pr